src/gui/gui.cpp

```diff
@@ -555,7 +555,7 @@ bool Gui_LoadScreenAssignPic(const char* pic_name)
     size_t pic_len = strlen(pic_name);
     size_t base_len = strlen(Engine_GetBasePath());
     size_t buf_len = pic_len + base_len + 5;
-    char image_name_buf[buf_len];
+    char *image_name_buf = new char[buf_len];
```

Comment:
1) Where is delete[]?
2) char *image_name_buf = (char*)Sys_GetTempMem(buf_len);

Reply:
hmm can be that the problem to some small freeze in game ? :x